samples/create-batch-prediction-job-video-classification.js

```diff
@@ -0,0 +1,249 @@
+      oneSecIntervalClassification: true,
+    });
+
+    const modelParameters = modelParamsObj.toValue();
+
+    const inputConfig = {
+      instancesFormat: 'jsonl',
+      gcsSource: {uris: [gcsSourceUri]},
+    };
+    const outputConfig = {
+      predictionsFormat: 'jsonl',
+      gcsDestination: {outputUriPrefix: gcsDestinationOutputUriPrefix},
+    };
+    const batchPredictionJob = {
+      displayName: batchPredictionDisplayName,
+      model: modelName,
+      modelParameters,
+      inputConfig,
+      outputConfig,
+    };
+    const request = {
+      parent,
+      batchPredictionJob,
+    };
+
+    // Create batch prediction job request
+    const [response] = await jobServiceClient.createBatchPredictionJob(request);
+
+    console.log(`Create batch prediction job video classification response`);
+    console.log(`\tName : ${response.name}`);
+    console.log(`\tDisplay name: ${response.displayName}`);
+    console.log(`\tModel : ${response.model}`);
+    console.log(`\tModel parameters : ${response.modelParameters}`);
+    console.log(`\tGenerate explanation : ${response.generateExplanation}`);
+    console.log(`\tState : ${response.state}`);
+    console.log(`\tCreate time : ${JSON.stringify(response.createTime)}`);
+    console.log(`\tStart time : ${JSON.stringify(response.startTime)}`);
+    console.log(`\tEnd time : ${JSON.stringify(response.endTime)}`);
+    console.log(`\tUpdate time : ${JSON.stringify(response.updateTime)}`);
+    console.log(`\tLabels : ${JSON.stringify(response.labels)}`);
+
+    const inputConfiguration = response.inputConfig;
+    console.log(`\tInput config`);
+    console.log(`\t\tInstances format : ${inputConfiguration.instancesFormat}`);
+
+    const gcsSource = inputConfiguration.gcsSource;
+    console.log(`\t\tGcs source`);
+    console.log(`\t\t\tUris : ${gcsSource.uris}`);
+
+    const bigquerySource = inputConfiguration.bigquerySource;
+    console.log(`\t\tBigQuery source`);
+    if (bigquerySource === null) {
+      console.log(`\t\t\tInput uri : {}`);
+    } else {
+      console.log(`\t\t\tInput uri : ${bigquerySource.inputUri}`);
+    }
+
+    const outputConfiguration = response.outputConfig;
+    console.log(`\t\tOutput config`);
+    console.log(
+      `\t\tPredictions format : ${outputConfiguration.predictionsFormat}`
+    );
+
+    const gcsDestination = outputConfiguration.gcsDestination;
+    console.log(`\t\tGcs destination`);
+    console.log(`\t\t\tOutput uri prefix : ${gcsDestination.outputUriPrefix}`);
+
+    const bigqueryDestination = outputConfiguration.bigqueryDestination;
+    if (bigqueryDestination === null) {
+      console.log(`\t\tBigquery destination`);
+      console.log(`\t\t\tOutput uri : {}`);
+    } else {
+      console.log(`\t\tBigquery destination`);
+      console.log(`\t\t\tOutput uri : ${bigqueryDestination.outputUri}`);
+    }
+
+    const dedicatedResource = response.dedicatedResource;
+    console.log(`\tDedicated resources`);
```

Comment:
There's a lot of logic here for the printing logic, I'd be tempted to see if I could abstract it a bit into a helper, that's also between the `START`/`END` blocks, something like:


```js
function printDedicatedResource(dedicatedResource) {
  
}

function printCompletionStats () {

}

...

```

If you moved these down to the bottom of the sample, I think you would draw better attention to the more important part of samples which is the calls to the API.

Reply:
Removed. Earlier versions of the Python canonicals had more verbose responses printed out. Now they're much shorter.